content-src/components/TopSites/TopSite.jsx

```diff
@@ -183,7 +183,9 @@ export class TopSiteLink extends React.PureComponent {
     }
     return (<li className={topSiteOuterClassName} onDrop={this.onDragEvent} onDragOver={this.onDragEvent} onDragEnter={this.onDragEvent} onDragLeave={this.onDragEvent} {...draggableProps}>
       <div className="top-site-inner">
-         <a href={link.searchTopSite ? undefined : link.url} tabIndex="0" onKeyPress={this.onKeyPress} onClick={onClick} draggable={true}>
+        {/*  Enhance accessibility of drag-and-drop anchor tags - see bug 1552005 */}
```

Comment:
I'd maybe word this as: "We don't yet support an accessible drag-and-drop implementation, see Bug 1552005"

Reply:
I'll adjust that!